src/components/SiteFormComponent.vue

```diff
@@ -1,105 +1,183 @@
+      (value: unknown) => (!isNaN(Number(value)) ? null : 'Muss eine Zahl sein'),
+    ],
   },
+  { name: 'street', label: 'Straße und Hausnummer', type: 'text' },
+  { name: 'city', label: 'Stadt', type: 'text' },
+  { name: 'zip', label: 'PLZ', type: 'text' },
+  { name: 'province', label: 'Bundesland', type: 'text' },
+  { name: 'country', label: 'Land', type: 'text' },
+];
+
+const initialValues = ref<Record<string, unknown>>(props.initialValues ?? {
+  title: '',
+  description: '',
+  usableSpace: '',
+  street: '',
+  city: '',
+  zip: '',
+  province: '',
+  country: '',
+});
+
+interface Tenant {
+  id: number;
+  firstName: string;
+  lastName: string;
+  period: string;
+  price: number;
+  deposit: number;
+  extraCosts: number;
+  email: string;
+}
+
+const items = ref<Tenant[]>([
   {
-    name: 'zip',
-    label: 'PLZ',
-    type: 'text',
-    required: false,
+    id: 1,
+    firstName: 'Lena',
+    lastName: 'Schneider',
+    period: '01.06.2025 - 11.06.2025',
+    price: 125,
+    deposit: 10,
+    extraCosts: 13,
+    email: 'lena.schneider@example.com',
   },
+]);
+
+const formerItems = ref<Tenant[]>([
   {
-    name: 'province',
-    label: 'Bundesland',
-    type: 'text',
-    required: false,
+    id: 2,
+    firstName: 'Tobias',
+    lastName: 'Keller',
+    period: '01.01.2023 - 31.12.2024',
+    price: 120,
+    deposit: 10,
+    extraCosts: 12,
+    email: 'tobias.keller@example.com',
   },
   {
-    name: 'country',
-    label: 'Land',
-    type: 'text',
-    required: false,
+    id: 3,
+    firstName: 'Miriam',
+    lastName: 'Fischer',
+    period: '01.01.2021 - 31.12.2023',
+    price: 115,
+    deposit: 9,
+    extraCosts: 12,
+    email: 'miriam.fischer@example.com',
   },
-];
+]);
 
-let initialValues = ref(props.initialValues);
+const showFormer = ref(false);
+// Backend-Aufruf auskommentiert
+/*
+onMounted(async () => {
+  const url = `/projects/${props.projectId}/sites/${props.siteId}/tenancies`;
+  try {
+    const res = await fetch(url);
+    const data = await res.json();
+    items.value = data.current || [];
+    formerItems.value = data.former || [];
+  } catch {
+    items.value = [];
+    formerItems.value = [];
+  }
+});
+*/
 
-if (!props.initialValues) {
-  initialValues = ref({
-    title: '',
-    description: '',
-    usableSpace: '',
-    street: '',
-    city: '',
-    zip: '',
-    province: '',
-    country: '',
-  });
-}
 </script>
 
 <template>
   <ReusableForm
-    :fields="fields"
-    :initialValues="initialValues"
-    :headline="headline"
-    :saveButtonText="saveButtonText"
-    :cancelButtonText="cancelButtonText"
-    :onSubmit="onSubmit"
-    :onCancel="onCancel"
+      :fields="fields"
+      :initialValues="initialValues"
+      :headline="headline"
+      :saveButtonText="saveButtonText"
+      :cancelButtonText="cancelButtonText"
+      :onSubmit="onSubmit"
+      :onCancel="onCancel"
   />
+  <div class="p-6 max-w-4xl mx-auto mt-10 shadow-lg bg-white rounded">
```

Comment:
```suggestion
  <div class="p-6 w-full mx-auto mt-10 shadow-lg bg-white rounded">
```
Anpassung der Klasse von max-w-4xl zu w-full für bessere Flexibilität im Layout.
vorher
<img width="1512" alt="Bildschirmfoto 2025-06-17 um 00 15 13" src="https://github.com/user-attachments/assets/9c8bc3be-c27b-4ef6-9fa0-39726632e4f4" />

nachher
<img width="1510" alt="Bildschirmfoto 2025-06-17 um 00 15 42" src="https://github.com/user-attachments/assets/8e528c9c-b44d-416f-9c07-f58408f21986" />

tabelle füllt nun den ganzen platz links rechts